server/protocols/jmap-rfc8621-integration-tests/pom.xml

```diff
@@ -35,6 +35,7 @@
     <modules>
         <module>jmap-rfc8621-integration-tests-common</module>
         <module>memory-jmap-rfc8621-integration-tests</module>
+        <module>distributed-jmap-rfc8621-integration-tests</module>
```

Comment:
you should respect the alphabetic ordering

Reply:
Yes, my bad.